### PR TITLE
vim-patch:4700398e384f

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/4700398e384f38f752b432e187462f404b96847d

partial skip:
doc/sign.txt
doc/various.txt
doc/motion.txt

### CODE REVIEW

File: runtime/doc/windows.txt [hunk 1]
```diff
@@ -1136,6 +1136,8 @@ list of buffers. |unlisted-buffer|
 		the current buffer remains being edited.  See |:buffer-!| for
 		[!].  This will also edit a buffer that is not in the buffer
 		list, without setting the 'buflisted' flag.
+		The notation with single quotes does not work here,
+		`:buf 12'345'` uses 12'345 as a buffer name.
```
Comment (seandewar) [inline runtime/doc/windows.txt:1140]: Feels scriptversion4 & vim9scriptey. I don't think we include this:
https://github.com/vim/vim/commit/af377e34b01ba00f9520d2b9de1f911e72db0114

File: runtime/doc/sign.txt [hunk 2]
```diff
@@ -416,11 +416,12 @@ sign_getdefined([{name}])				*sign_getdefined()*
...
 		   icon		full path to the bitmap file of the sign
 		   linehl	highlight group used for the whole line the
-				sign is placed in.
+				sign is placed in; not present if not set.
 		   name		name of the sign
 		   text		text that is displayed when there is no icon
 				or the GUI is not being used.
-		   texthl	highlight group used for the text item
+		   texthl	highlight group used for the text item; not
+			present if not set.
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/sign.txt:424]: Was like that in the original, and had me wondering, too ;)

**Review by seandewar — COMMENTED** [review]
Need syntax/vim.vim change

https://github.com/vim/vim/commit/4700398e384f38f752b432e187462f404b96847d#diff-7a82881606d3861674909df27974e1c38c3e031d00f486ca01ccff1d9bceed0f

**Comment by clason** [comment]
That looked to be in the autogenerated section?

**Comment by seandewar** [comment]
Yeah, that confused me too, but I'm pretty sure that should be updated manually

**Comment by clason** [comment]
Yeah, I think you're right. Added! ❤️

**Review by seandewar — APPROVED** [review]

**Pull request merged**